change/@fluentui-date-time-utilities-2020-07-08-09-37-29-fix-date-time-utils-tz-setup.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "none",
+  "comment": "Set timezone for tests via env variable",
```

Comment:
Should probably update this comment now?

Reply:
Yeah, good catch, thank you! :)